### ISSUE TITLE
[A11Y] In-context help on Title form field during new submission doesn't work on MacOs

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9355]
## Issue Description
// What is the nature of the issue? How does it impact users?

When focusing the title field using a screen reader, there is an instruction using aria-label that doesn't work on MacOS

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

1. Create a new submission
2. On the first page when you are inserting the title and using help of screenreader to navigate the page

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method
- Method: VoiceOver
- Automated test: Siteimprove Accessibility Checker

## Action
// Describe action that can be taken to remediate the issue. 

Create and test a help shortcut using keyboard that works on Windows, MacOS and Linux. 

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

body id="tinymce" class="mce-content-body " data-id="titleAbstract-title-control-en" aria-label="Rich Text Area. Press ALT-0 for help."

**Comment by jardakotesovec** [comment]
Related ticket: https://github.com/pkp/pkp-lib/issues/12721

WIP PR:
ojs (deployment): https://github.com/pkp/ojs/pull/5852
pkp-lib: https://github.com/pkp/pkp-lib/pull/13374
ui-library: https://github.com/pkp/ui-library/pull/991

Deployed as pr-5852
